<?xml version="1.0" ?>
<launch>
  <node pkg="industrial_extrinsic_cal" type="target_locator_srv" name="target_locator" output="screen" >
    <param name=" image_topic" value="Asus/rgb/image_rect_color" />
    <param name="target_rows" value="4" />
    <param name=" target_cols" value= "4" />
    <param name="target_circle_dia" value="0.0127" />
    <!--param name="target_spacing" value="0.0209903" /-->
    <param name="target_spacing" value="0.06297168" />
    <param name="focal_length_x" value="525.0" />
    <param name="focal_length_y" value="525.0" />
    <param name="center_x" value="320.0" />
    <param name="center_y" value="210.0" />
  </node>

  <node pkg="industrial_extrinsic_cal" type="call_service" name="call_service" output="screen" />

  <include  file="$(find industrial_extrinsic_cal)/launch/single_sensor.launch">
	<arg name="arm_prefix" value="asus"/>
	<arg name="camera_name" value="Asus"/>
	<arg name="publish_tf" value="false" />
	<arg name="device_id" value="2@0" />
  </include> 

  <node pkg="image_view" type="image_view" name="viewit" >
        <remap from="image" to="Asus/rgb/image_rect_color"/>
  </node>

  <node pkg="image_view" type="image_view" name="debug_view" >
        <remap from="image" to="observer_raw_image"/>
  </node>

</launch>
